DESCRIPTION.md

```diff
@@ -8,6 +8,9 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
 
 # Release Notes
 
+- v3.3.0(Unreleased)
+  - Updated to use apache arrow-nanoarrow project for result arrow data conversion.
```

Comment:
Maybe "Updated to use Apache ... for processing query results in Arrow format"?
Btw does this affect `write_pandas`? Pandas has a required dependency on `PyArrow`.

Reply:
it will not affect write_pandas feature, pandas & pyarrow are still optional/extra requirements as usual